crates/aiken-project/src/docs.rs

```diff
@@ -268,17 +268,40 @@ fn generate_module(
         timestamp: timestamp.as_secs().to_string(),
     };
 
+    let rendered_content = convert_latex_markers(
+                                    inject_math_library(
+                                        module.render().expect("Module documentation template rendering"),
+                                        )
+                                    );
+
     (
         search_indexes,
         DocFile {
             path: PathBuf::from(format!("{}.html", module.module_name)),
-            content: module
-                .render()
-                .expect("Module documentation template rendering"),
+            content: rendered_content,
         },
     )
 }
 
+
+fn convert_latex_markers(input: String) -> String {
+    input.replace("#[", "\\(")
+         .replace("]#", "\\)")
+}
+
+fn inject_math_library(html: String) -> String {
+    let mathjax_script = r#"
+    <script type="text/javascript" id="MathJax-script" async
+        src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js">
+    </script>
+    <script>
+      MathJax.typeset();
+    </script>
+    "#;
+
+    html.replace("</head>", &format!("{}\n</head>", mathjax_script))
+}
+
```

Comment:
Hey, we probably don't need to inject this from the rust side, but can directly add it to the doc template: https://github.com/aiken-lang/aiken/blob/main/crates/aiken-project/templates/_layout.html#L3-L12